src/Roslyn.Suppressions/Roslyn.Suppressions/Settings.Cache/SettingsCache.cs

```diff
@@ -42,17 +43,18 @@ internal SettingsCache(IRoslynSettingsFileStorage fileStorage, ConcurrentDiction
 
         public RoslynSettings GetSettings(string settingsKey)
         {
-            if(!settingsCollection.ContainsKey(settingsKey))
+            var normalisedKey = NormalizeKey(settingsKey);
```

Comment:
A better approach is to create the dictionary with type of key comparer we want it to use i.e.

```
new ConcurrentDictionary<string, RoslynSettings>(StringComparer.OrdinalIgnoreCase)
```

It's more reliable than remember to manually convert the key every time (and more efficient, since it doesn't involve converting and creating a new string.

Also, the MS Docs [recommend](https://docs.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings#choosing-a-stringcomparison-member-for-your-method-call) using `OrdinalIgnoreCase` when comparing file paths.